test/test_parmed_amber.py

```diff
@@ -1600,6 +1601,14 @@ def test_auto_detection(self):
             f.write('Only one line\n')
         self.assertFalse(asciicrd.AmberAsciiRestart.id_format(fn))
 
+    def test_parsing_netcdf_file_without_unitcells(self):
```

Comment:
This test needs a `unittest.skipIf(PYPY, 'Cannot run test under PYPY')`.  Or something like that -- the other NetCDF tests in test_parmed_amber.py have examples.